### PR TITLE
[Demo] Fix diverging stacked bar in ViViVo

### PR DESCRIPTION
**Pull request by huong-li-nguyen**
## Description
- Fix order of traces to be consistent with legend order
- A couple of open questions / thoughts I would like to discuss later @antonymilne 

## Screenshot

Before:

I would recommend this pastry to my friends

After:

386191622-8bbb1ce8-4ec5-4244-a940-b19cadc9942c

## Notice

- [x] I acknowledge and agree that, by checking this box and clicking "Submit Pull Request":

  - I submit this contribution under the [Apache 2.0 license](https://www.apache.org/licenses/LICENSE-2.0.txt) and represent that I am entitled to do so on behalf of myself, my employer, or relevant third parties, as applicable.
  - I certify that (a) this contribution is my original creation and / or (b) to the extent it is not my original creation, I am authorized to submit this contribution on behalf of the original creator(s) or their licensees.
  - I certify that the use of this contribution as authorized by the Apache 2.0 license does not violate the intellectual property rights of anyone else.
  - I have not referenced individuals, products or companies in any commits, directly or indirectly.
  - I have not added data or restricted code in any commits, directly or indirectly.

### CODE REVIEW

File: vizro-core/examples/scratch_dev/app.py [hunk 1]
```diff
@@ -23,21 +25,100 @@
...
+    fig.update_layout(barmode="relative")
+    fig.add_vline(x=0, line_width=2, line_color="grey")
+
+    # Update legend order to go from most negative to most positive
+    category_order = category_neg[::-1] + category_pos
+    for i, category in enumerate(category_order):
+        fig.update_traces(legendrank=i, selector=({"name": category}))
+
+
+    fig.update_xaxes(ticksuffix="%", range=[-80, 80])
```
Comment (huong-li-nguyen) [inline vizro-core/examples/scratch_dev/app.py:100]: This previously worked fine, but now it currently does not work anymore 🤔 It just changes the range of the positive side. Is there another way of doing this now?

File: vizro-core/examples/visual-vocabulary/pages/examples/diverging_stacked_bar.py [hunk 2]
```diff
@@ -63,6 +75,6 @@ def diverging_stacked_bar(data_frame: pd.DataFrame, **kwargs) -> go.Figure:
     data_frame=pastries,
     x=["Strongly Disagree", "Disagree", "Agree", "Strongly Agree"],
     y="pastry",
-    labels={"value": "Response count", "variable": "Opinion"},
+    labels={"value": "", "variable": "", "pastry": ""},
```
Comment (huong-li-nguyen) [inline vizro-core/examples/visual-vocabulary/pages/examples/diverging_stacked_bar.py:77]: Declutter chart - axes labels and legend titles are redundant here

File: vizro-core/examples/visual-vocabulary/custom_charts.py [hunk 3]
```diff
@@ -303,4 +303,5 @@ def diverging_stacked_bar(data_frame: pd.DataFrame, **kwargs) -> go.Figure:
     else:
         fig.add_hline(y=0, line_width=2, line_color="grey")
 
+    fig.update_xaxes(ticksuffix="%", range=[0, 100])
```
Comment (huong-li-nguyen) [inline vizro-core/examples/visual-vocabulary/custom_charts.py:306]: @antonymilne to double-check range

Comment (huong-li-nguyen) [reply vizro-core/examples/visual-vocabulary/custom_charts.py:306]: Yes, agree on the 100 mark! Oh wow, thanks for figuring this out! 🧠 ⭐

**Review by petar-qb — APPROVED** [review]
I like these two bugfixes!! The chart is much more readable now 🎨 
May you update the PR description with the latest version of this chart? 

image

**Pull request merged**